src/Avalonia.Visuals/Media/HsvColor.cs

```diff
@@ -0,0 +1,493 @@
+
+        /// <summary>
+        /// Creates a new <see cref="HsvColor"/> from individual color channel values.
+        /// </summary>
+        /// <remarks>
+        /// This exists for symmetry with the <see cref="Color"/> struct; however, the
+        /// appropriate constructor should commonly be used instead.
+        /// </remarks>
+        /// <param name="a">The Alpha (transparency) channel value in the range from 0..1.</param>
+        /// <param name="h">The Hue channel value in the range from 0..360.</param>
+        /// <param name="s">The Saturation channel value in the range from 0..1.</param>
+        /// <param name="v">The Value channel value in the range from 0..1.</param>
+        /// <returns>A new <see cref="HsvColor"/> built from the individual color channel values.</returns>
+        public static HsvColor FromAhsv(double a, double h, double s, double v)
+        {
+            return new HsvColor(h, s, v, a);
+        }
+
+        /// <summary>
+        /// Creates a new <see cref="HsvColor"/> from individual color channel values.
+        /// </summary>
+        /// <remarks>
+        /// This exists for symmetry with the <see cref="Color"/> struct; however, the
+        /// appropriate constructor should commonly be used instead.
+        /// </remarks>
+        /// <param name="h">The Hue channel value in the range from 0..360.</param>
+        /// <param name="s">The Saturation channel value in the range from 0..1.</param>
+        /// <param name="v">The Value channel value in the range from 0..1.</param>
+        /// <returns>A new <see cref="HsvColor"/> built from the individual color channel values.</returns>
+        public static HsvColor FromHsv(double h, double s, double v)
+        {
+            return new HsvColor(h, s, v, 1.0);
+        }
+
+        /// <summary>
+        /// Converts the given HSV color to it's RGB color equivalent.
+        /// </summary>
+        /// <param name="hsvColor">The color in the HSV color model.</param>
+        /// <returns>A new RGB <see cref="Color"/> equivalent to the given HSVA values.</returns>
+        public static Color ToRgb(HsvColor hsvColor)
+        {
+            return HsvColor.ToRgb(hsvColor.H, hsvColor.S, hsvColor.V, hsvColor.A);
+        }
+
+        /// <summary>
+        /// Converts the given HSVA color channel values to it's RGB color equivalent.
+        /// </summary>
+        /// <param name="hue">The hue channel value in the HSV color model in the range from 0..360.</param>
+        /// <param name="saturation">The saturation channel value in the HSV color model in the range from 0..1.</param>
+        /// <param name="value">The value channel value in the HSV color model in the range from 0..1.</param>
+        /// <param name="alpha">The alpha channel value in the range from 0..1.</param>
+        /// <returns>A new RGB <see cref="Color"/> equivalent to the given HSVA values.</returns>
+        public static Color ToRgb(
```

Comment:
Color model conversion code is well tested in the C# ports of the WinUI color picker and then again with this HsvColor struct on an internal project. This code is working fine in a real ColorPicker.

Reply:
Marking resolved since no additional comments in review